src/components/filter/FilterBoilerplate.vue

```diff
@@ -213,13 +213,6 @@ export default {
       type: Boolean,
       default: true
     },
-    /**
-     * Display the filter on dark background
-     */
-    dark: {
```

Comment:
We should keep the `dark` property on this component (with the CSS you already removed) but pass this property down to `SearchFormControl`. This way the filter can also be display in its modal (non dark).

So the properties should be written:

```
:rounded="false"
:dark="dark"
```

Reply:
Alright thanks, I was wondering if this was still used, I'll add it back